src/operator/tensor/broadcast_reduce_op_value.cc

```diff
@@ -68,9 +56,7 @@ MXNET_OPERATOR_REGISTER_REDUCE_BACKWARD(_backward_prod)
 .set_attr<FCompute>("FCompute<cpu>", ReduceAxesBackwardUseInOut< cpu, mshadow_op::rdiv>);
 
 MXNET_OPERATOR_REGISTER_REDUCE(nansum)
-.describe(R"code(Compute the sum of array elements over given axes with ``NaN`` ignored
-
-Refer to ``sum`` for more details.
+.describe(R"code(Compute the sum of array elements over given axes treating Not a Numbers ``NaNs`` as zeros.
```

Comment:
the "s" cannot be inside the double backticks. Just put "treating each ``NaN`` as a zero."